lib/semantics/mod-file.cc

```diff
@@ -41,10 +41,10 @@ using namespace parser::literals;
 // that the module file is decoded as UTF-8 even if source files
 // are using another encoding.
 struct ModHeader {
-  static constexpr const char *bom{"\xef\xbb\xbf"};
-  static constexpr const char *magic{"!mod$ v1 sum:"};
+  static constexpr const char bom[3 + 1]{"\xef\xbb\xbf"};
   static constexpr int magicLen{13};
-  static constexpr int sumLen{16};
+  static constexpr int sumLen{magicLen + 3};
```

Comment:
`sumLen` is not related to `magicLen`, it's just the number of characters in the checksum.

Reply:
Sure; thanks.